typescript-deno/README.md

```diff
@@ -0,0 +1,33 @@
+# TypeScript
+
+Skeleton project for TypeScript with testing via [Deno](https://deno.land/).
+
+Make sure you have Deno installed by running `deno --version`.
+
+If using VSCode, you might want to enable the extension in the settings:
+
+```jsonc
+// .vscode/settings.json
+{
+  "deno.enable": true
+}
+```
+
+## Usage
+
+- `deno run src/mod.ts` to run your code
```

Comment:
We could put these commands into  ./scripts/start and ./scripts/test files to match other repos?